android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java

```diff
@@ -69,7 +69,7 @@
  * 4. Helper methods
  */
 public class SdlManager extends BaseSdlManager {
-    private Context context;
+    Context context;
```

Comment:
@JulianKast can't process this comment, otherwise, there will be no context access in BaseManager class which requires it to get vehicles from Manifest

Reply:
@kostyaBoss you do not need the context. The implementation is incorrect according to the proposal. You should also **never** access a variable stored in a child class from a parent class without even checking if the instance in question is actually of the child class. The code that accesses this variable causes an instance exception in JavaSE which should have been caught if this was actually tested against JavaSE as you have stated. Please follow all of these requests as provided. Thank you.

Reply:
@joeygrover Thanks for your reply. I am confused, because in the BaseSdlManager we should process the systemInfo and manifest specific things. This is not possible without having the context reference (the screenshot is attached). On the other hand I didn't find any place where I can get the context from. Not sure If I can move context out of there
<img width="840" alt="изображение" src="https://user-images.githubusercontent.com/20925367/106926597-eb08d500-6719-11eb-919a-dd201db3484a.png">

Reply:
That is incorrect. The proposal states the developer is responsible for this exact logic. If you follow the review from @JulianKast you will see that the code here does not follow the proposal and creates disparity between the other platforms. The system info is handled by the developer through the listener they pass to the SdlManager. Please read through the entire review to understand where the issues are and address them as a whole. Thank you.